src/gemm.jl

```diff
@@ -54,53 +56,68 @@ end
     VectorizationBase.OffsetPrecalc(notropical(ptr.ptr), ptr.precalc)
 end
 
-@inline function VectorizationBase._vload(ptr::AbstractStridedPointer{Tropical{T}}, u::Unroll, a::A, si::StaticInt{RS}) where {T,A<:StaticBool,RS}
-    res = VectorizationBase._vload(notropical(ptr), u, a, si)
+@inline function VectorizationBase._vload(ptr::AbstractStridedPointer{Tropical{T}}, u::Unroll, ::A, ::StaticInt{RS}) where {T,A<:StaticBool,RS}
+    res = VectorizationBase._vload(notropical(ptr), u, A(), StaticInt{RS}())
```

Comment:
I always wonder what is the difference between passing an argument and writing the constructor explicitly?